portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -45,10 +47,12 @@ public Comment(String email, String comment, String date){
   }
   private DatastoreService datastore;
   private static final String COMMENT_DATE_FORMAT = "MMM dd,yyyy HH:mm";
+  private static PolicyFactory sanitizer;
```

Comment:
You don't want `static` for this unless you're willing to initialize it on this line (and make it final).